pkg/kubelet/dockertools/labels.go

```diff
@@ -45,14 +49,37 @@ func newLabels(container *api.Container, pod *api.Pod, restartCount int) map[str
 	labels := map[string]string{}
 	labels[kubernetesPodNameLabel] = pod.Name
 	labels[kubernetesPodNamespaceLabel] = pod.Namespace
-	labels[kubernetesPodUID] = string(pod.UID)
+	labels[kubernetesPodUIDLabel] = string(pod.UID)
 
+	labels[kubernetesContainerNameLabel] = container.Name
+	labels[kubernetesContainerHashLabel] = strconv.FormatUint(kubecontainer.HashContainer(container), 16)
 	labels[kubernetesContainerRestartCountLabel] = strconv.Itoa(restartCount)
-	labels[kubernetesContainerTerminationMessagePath] = container.TerminationMessagePath
+	labels[kubernetesContainerTerminationMessagePathLabel] = container.TerminationMessagePath
 
 	return labels
 }
 
+// TODO: (random-liu) When old containers are deprecated, we should change the following `Info` log to `Error` log, or return error here
```

Comment:
Agreed. I think it'd be better to convert all the kubelet-managed labels and return a single object.